src/jit/flowgraph.cpp

```diff
@@ -4261,33 +4261,6 @@ void        Compiler::fgFindJumpTargets(const BYTE * codeAddr,
         pSm->Start(this);
     }
 
-#ifdef FEATURE_LEGACYNETCF
```

Comment:
+1. I noticed this as well when I was working on the extended JIT flags--not sure if it was intentional, but the symbol is defined by default in our current builds.